api/src/main/java/org/openmrs/User.java

```diff
@@ -53,14 +71,74 @@
 	
 	private String email;
 	
+<<<<<<< HEAD
+	@ManyToOne(optional = false)
+	@JoinColumn(name = "creator")
+	private User creator;
+	
+	@Column(name = "date_created", nullable = false)
+	private Date dateCreated;
+	
+	@ManyToOne
+	@JoinColumn(name = "changed_by")
+	private User changedBy;
+	
+	@Column(name = "date_changed")
+	private Date dateChanged;
+	
+	@Column(name = "retired", nullable = false)
+	private String retired;
+	
+	@ManyToOne
+	@JoinColumn(name = "retired_by")
+	private User retiredBy;
+	
+	@Column(name = "retire_reason", length = 255)
+	private String retireReason;
+	
+	@Column(name = "date_retired")
+	private Date dateRetired;
+	
+	@SuppressWarnings("deprecation")
+	@ManyToMany
+	@JoinTable(name = "user_role", joinColumns = @JoinColumn(name = "user_id"), inverseJoinColumns = @JoinColumn(name = "role"))
+	@LazyCollection(LazyCollectionOption.FALSE)
+	@Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
+	@Cascade({ CascadeType.SAVE_UPDATE, CascadeType.MERGE, CascadeType.EVICT })
 	private Set<Role> roles;
 	
+	@SuppressWarnings("deprecation")
+	@ElementCollection
+	@CollectionTable(name = "user_property", joinColumns = @JoinColumn(name = "user_id", nullable = false))
+	@MapKeyColumn(name = "property", length = 100)
+	@Column(name = "property_value")
+	@Cascade({ CascadeType.SAVE_UPDATE, CascadeType.MERGE, CascadeType.EVICT })
+=======
+	private Set<Role> roles;
+	
+>>>>>>> 7f1edc92d80496f5d1c86e9e0b968292c5ebf238
```

Comment:
Here we have something similar. However, unlike the above, we don't just cleanly want option 1 or option 2. We primarily want what's in option 2, but we also want the fields (but not the annotations) defined in option 1. 

If you want to understand *why* you get this conflict, it's because the version of `master` you were working from had [this commit](https://github.com/openmrs/openmrs-core/commit/267f4df2ba3d683323e2938690f5bcef116318b6#diff-7325e970b68864df576bcb7c49e495c0), but it's [since been backed out](https://github.com/openmrs/openmrs-core/commit/4d53a02710df2fda97ca5579772b629022267677#diff-7325e970b68864df576bcb7c49e495c0); this is also why we want option 2 with some additions. While we don't want to add all the Hibernate annotations in, we do want the fields that used to be defined in `BaseChangeableOpenmrsMetadata` that are required to support the `Auditable` adn `Retireable` interfaces.

So in this case, we want to end up with something like this:

```suggestion
	private User creator;
	
	private Date dateCreated;

	private User changedBy;
	
	private Date dateChanged;
	
	private String retired;
	
	private User retiredBy;
	
	private String retireReason;
	
	private Date dateRetired;

	private Set<Role> roles;
```